dynoid/dynoidtest/dynoidtest.go

```diff
@@ -44,6 +59,10 @@ func (iss *Issuer) GenerateIDToken(clientID string) (string, error) {
 	return token.SignedString(iss.key)
 }
 
+func (iss *Issuer) Context() context.Context {
+	return iss.ctx
+}
+
```

Comment:
This feels a bit dirty and seems to only be necessary for the test code. Can we pass a `context.Background()` through there to make the interface smaller?

Reply:
@tt I refactored it a bit to get rid of that. Let me know what you think.